source/_docs/tools/dev-tools.markdown

```diff
@@ -24,3 +24,79 @@ Screenshot of Home Assistant's Developer Tools.
 | Templates | <img src='/images/screenshots/developer-tool-templates-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> | Renders templates |
 | Info | <img src='/images/screenshots/developer-tool-about-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> | Details about Home Assistant |
 
+## What can I do with Developer Tools?
+The Developer Tools is meant for **all** (not just for the developers) to quickly try out things - like calling services, update states, raising events, and publish messages in mqtt…etc.). It is also a necessary tool for those who write custom automations and scripts by hand. The following describes each of the section in detail.
+
+{% linkable_title Services %}
+
+This section is used to call Services that are available in the ServiceRegistry.
```

Comment:
Is it `ServiceRegistry` or `Service Registry`?

Reply:
I wanted to use `ServiceRegistry` as one word, as it also indicates the `ServiceRegistry` component. It resonates well with "real" developers, as well as with non-developer community.